client/packages/common/src/intl/locales/en/common.json

```diff
@@ -478,6 +478,8 @@
   "heading.order-info": "Order info",
   "heading.password": "Password",
   "heading.patient-details": "Patient Details",
+  "label.patient-name": "Patient name",
+  "label.patient-weight": "Patient weight",
```

Comment:
We'll need this one eventually, but I think it's probalby still good practice to just add the strings we actually need in the PR, not additional ones for future use, as they might end up duplicated, or creating translation work that doesn't need to be done.